pkg/cfn/manager/api.go

```diff
@@ -167,8 +167,7 @@ func (c *StackCollection) DescribeStack(i *Stack) (*Stack, error) {
 	return resp.Stacks[0], nil
 }
 
-// ListStacks gets all of CloudFormation stacks
-func (c *StackCollection) ListStacks(nameRegex string, statusFilters ...string) ([]*Stack, error) {
+func (c *StackCollection) listStacks(nameRegex string, statusFilters ...string) ([]*Stack, error) {
```

Comment:
Could you please export this as something like `ListStacksMatching`? It's handy when you want to discover stacks you have, started working on cluster deleter lambda the other day, and there I used `ListStacks("eksctl-.*")`.

Reply:
Yup, makes sense. Will do.

Reply:
Ping!

Reply:
Re-added a `ListStacksMatching` function.